doc/Language/statement-prefixes.pod6

```diff
@@ -80,16 +80,18 @@ don't care.
 
 =head2 X<C<quietly>|Syntax,quietly (statement prefix)>
 
-As a statement prefix, C<quietly> suppresses all warnings produced by the
-statement it precedes.
+As a prefix, C<quietly> suppresses all warnings produced by the
+block or statement it precedes (currently only run-time warnings).
 
 =for code
 sub marine() {};
 quietly say ~&marine; # OUTPUT: «marine␤»
+sub told-you { warn 'hey...' }; 
+quietly { told-you; warn 'kaput!' };
+warn 'Telling you now!';  # OUTPUT: «Telling you now!␤ [...] ␤»
 
 Calling L<C<.Str> on C<code> produces a warning|/type/Code#method_Str>.
-Preceding the statement with C<quietly > will just produce the output, the name
-of the routine.
+Preceding the code with C<quietly > will just produce the output without warning.
 
```

Comment:
Can you remove the extra space in the C<>? I know it was there before, thanks.

Reply:
Done.  @coke, thanks twice.